README.rst

```diff
@@ -75,12 +75,24 @@ To see how a Momentum ResNet can be created using a ResNet, you can run:
 .. code:: python
 
    >>> import torch
-   >>> from momentumnet import transform
+   >>> from momentumnet import transform_to_momentumnet
    >>> from torchvision.models import resnet101
-   >>> mresnet101 = transform(resnet101(), gamma=0.99)
+   >>> mresnet101 = transform_to_momentumnet(resnet101(), gamma=0.99, use_backprop=False)
```

Comment:
split it in two lines: `resnet = resnet101(pretrained=True)`, then mresnet = ...